### ISSUE TITLE
use angular bootstrap modal for insert link (edit link)

### ISSUE THREAD

**Issue by mangrovestudios** [fraywing/textAngular#810]
Following up on issue #419.

Using modals end to end, how would the modal work in the onElementSelect block? I tried this:

```javascript
reLinkButton.on('click', function(event){
   event.preventDefault();
	
   var modalInstance = $modal.open({
      templateUrl : '/templates/dialog/linkMaker.html',
      controller  : 'linkMakerDlgCtrl',
      resolve     : {
         // variables here ...
         }
      }
   });
   modalInstance.result.then(function (link) {

     $element.attr('href', 'http://' + link.ref);
     editorScope.updateTaBindtaTextElement();

   }, function () { $log.debug('Modal dismissed'); });

   editorScope.hidePopover();
});
```

Does it also need the promise and return false? The problem with the above appears to be with `editorScope.updateTaBindtaTextElement();` in that the `$element` attribute is set but not applied to DOM.

**Comment by gimm** [comment]
Did you get it to work? I had similar problems with image insert with modal.

**Comment by mangrovestudios** [comment]
Not yet. I'm still stuck on this one..

**Comment by mangrovestudios** [comment]
Solved by updating textAngular version from 1.2.2 to latest 1.4.3
